### ISSUE TITLE
Biome DFU

### ISSUE THREAD

**Issue by IllagerCaptain** [SkyblockerMod/Skyblocker#723]
### Describe the new feature.

Something I noticed is biomes on SkyBlock are really janked up in newer versions. They appear to stay somewhat correctly shaped, not completely though as that is impossible in newer versions. They are also quite offset, most likely due to the same issue of biomes being stored in batches of blocks instead of per block in newer versions. They also sometimes use the wrong biome, for example, dark oak forest appearing as windswept forest. The wrong biome issue is just a Hypixel issue though and should be easy for _them_ to fix, but overall because some of these issues are due to biomes being stored differently, some of these issues are either not possible or too hard for Hypixel to bother with to fix. So I suggest SkyBlocker to enable the 1.8 biome system while the player is in SkyBlock.

**Comment by Fluboxer** [comment]
https://hypixel.net/bug-reports/create

**Comment by IllagerCaptain** [comment]
> https://hypixel.net/bug-reports/create

Reason I'm reporting here is that biomes are completely different between version and some of these issues cannot be fixed without mods.

**Comment by viciscat** [comment]
> Reason I'm reporting here is that biomes are completely different between version and some of these issues cannot be fixed without mods.

Items and countless other things are also completely different and hypixel does it just fine.

> And SkyBlocker also has an item DFU

I might be wrong but I'm pretty sure the DFU is only for NEU's item-repo

**Comment by IllagerCaptain** [comment]
> Items and countless other things are also completely different and hypixel does it just fine.
Most of the time when Minecraft changes functionality it adds new options. The issue here is that when biomes added to support 3D, they lost precision. In 1.8 biomes were per column of blocks, but now they are batches of blocks. While Hypixel can somewhat fix it by fixing the offset, this won't fix it completely as that is impossible with the current biome system.

**Comment by IllagerCaptain** [comment]
For example, in newer versions when you attempt to set a rectangle to a specific biome, this happens:
<img src="https://minecraft.wiki/images/Fillbiome_example-after.png?19ef9&format=original" />

**Comment by viciscat** [comment]
yeah biome data is indeed less precise, then biome blending does it's things. But that's still an hypixel issue in my humble imho opinion. Depends on the other devs' opinions.
